server/dba/seeders/prod/01-forests.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
```

Comment:
this should be updated to reflect that is just for christmas trees as this information will not be useful to other permit types. We may want to consider changing the IDs to be the actual national service IDs which we can provide for you.

Reply:
@lauraGgit we have added `org_structure_code` column to this table in sprint-2. Let's discuss this and come up with a plan to share or not share this table.

Reply:
@lpendem please make a story to address it.

Reply:
@lauraGgit story created https://github.com/nciinc/fs-intake-module/issues/79